meta-ostro/conf/distro/include/ostroproject-ci.inc

```diff
@@ -44,14 +48,25 @@ OSTRO_VM_IMAGE_TYPES = "dsk.xz dsk.zip dsk.vdi.zip"
 # which must contain only alphanumeric symbols,'-' and '_'.
 # Any other symbols would be skipped in parser.
 #
-# Following targets would be used to perform default build task:
-OSTROPROJECT_CI_BUILD_TARGETS="${DISTRO}-image ${DISTRO}-image-dev"
+# Following targets would be used to perform default build task.
+OSTROPROJECT_CI_BUILD_TARGETS="ostro-image ostro-image-qa ostro-image-reference"
 # Following targets would be executed with do_populate_sdk task
 OSTROPROJECT_CI_SDK_TARGETS=""
-# Following targets would be executed with do_populate_sdk_ext task
-OSTROPROJECT_CI_ESDK_TARGETS="${DISTRO}-image"
-# Following targets would be executed with do_test_iot_export task
-OSTROPROJECT_CI_TEST_EXPORT_TARGETS="${DISTRO}-image"
+# Following targets would be executed with do_populate_sdk_ext task.
+# Uses the fully-populated image to ensure that the eSDK is as complete
+# as possible.
+OSTROPROJECT_CI_ESDK_TARGETS="ostro-image-all"
+# Following targets would be executed with do_test_iot_export task.
+# Must be "ostro-image", the task seems to expect that name.
+OSTROPROJECT_CI_TEST_EXPORT_TARGETS="ostro-image"
+
+# ostro-image as build target above shall only produce bundles,
+# but no images.
+OSTRO_VM_IMAGE_TYPES_pn-ostro-image = ""
+IMAGE_FSTYPES_pn-ostro-image = ""
```

Comment:
On Tue, 2016-03-29 at 01:21 -0700, Alexander D. Kanevskiy wrote:
> > +# ostro-image as build target above shall only produce bundles,
> > +# but no images.
> > +OSTRO_VM_IMAGE_TYPES_pn-ostro-image = ""
> > +IMAGE_FSTYPES_pn-ostro-image = ""
> 
> should this go to ostro-image.bb recipe ?

No, because it only applies when building bundles in the CI. A developer
building ostro-image without swupd needs the original IMAGE_FSTYPES.